speakeasy/windows/fileman.py

```diff
@@ -235,6 +252,11 @@ def get_pipe_from_handle(self, handle):
         return self.pipe_handles.get(handle)
 
     def get_file_from_path(self, path):
+        # The emulated sample is requesting itself. The module path
+        # for it is(?) always the first entry in self.files
```

Comment:
thanks for the comment here. maybe add a similar comment to the property declaration in the ctor?